packages/lesswrong/components/common/HomeEA.jsx

```diff
@@ -20,7 +21,12 @@ class HomeEA extends PureComponent {
         }
 
         <HomeLatestPosts />
-        <RecentDiscussionThreadsList terms={{view: 'recentDiscussionThreadsList', limit:20}}/>
+        <RecentDiscussionThreadsList
+          terms={{view: 'recentDiscussionThreadsList', limit:20}}
+          commentsLimit={recentDiscussionCommentsPerPost}
+          maxAgeHours={18}
+          af={false}
```

Comment:
I'd have "af" be false by default, no reason every forum should need to specifiy this

Reply:
`af` isn't going to `RecentDiscussionThreadsList`, it's going to its `withMulti` HoC, which doesn't have a mechanism for specifying default values.